.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -2,6 +2,10 @@
 
 ## What is the value of this and can you measure success?
 
+<!--
+If setting up an AB test, make sure you have read our [AB testing doc](https://github.com/guardian/frontend/blob/master/docs/03-dev-howtos/01-ab-testing.md).
```

Comment:
> People may have been unable to set up an AB test

Do you mean able? 

I agree though. A summary is good. Maybe we can just make the first part of this better: https://github.com/guardian/frontend/blob/master/docs/03-dev-howtos/01-ab-testing.md#guide

What do you think?